UI/landing-page/style.css

```diff
@@ -0,0 +1,69 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+section{
+    background-color:#252934;
+    /* min-height: 100vh; */
```

Comment:
remove this comment line if it has no use.

Reply:
Removed, thanks. But I want to ask if it is allowed to include the comments, thanks!

Reply:
The comments are allowed only if it makes your code readable.